extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -1327,6 +1327,101 @@ describe('Project: properties', function (): void {
 		should(() => project.getDatabaseDefaultCollation())
 			.throw('Invalid value specified for the property \'DefaultCollation\' in .sqlproj file');
 	});
+
+	it('Should add database source to project property', async function (): Promise<void> {
```

Comment:
Is it valid to _add_ database sources with semicolons in the name?  I'm fine if the answer is "no", but this should validate that an error is thrown in that case.

Reply:
Might be missing it, but I'm not seeing a case that confirms negative or positive functionality for names containing a semicolon.

Reply:
I added a test case `Should throw error when adding or removing database source that contains semicolon`, is this what you're looking for?

Reply:
Aha, that's the one.  Appreciate it